client/src/test/java/io/split/storages/pluggable/synchronizer/TelemetryConsumerSubmitterTest.java

```diff
@@ -48,6 +53,27 @@ public void testTestSynchronizeConfig() throws NoSuchFieldException, IllegalAcce
         modifiersField.setInt(telemetryConsumerSubmitterHolder, telemetryConsumerSubmitterHolder.getModifiers() & ~Modifier.FINAL);
         telemetryConsumerSubmitterHolder.set(telemetrySynchronizer, safeUserStorageWrapper);
         telemetrySynchronizer.synchronizeConfig(splitClientConfig, 10L, new HashMap<>(), new ArrayList<>());
-        Mockito.verify(safeUserStorageWrapper, Mockito.times(1)).set(Mockito.anyString(), Mockito.anyObject());
+        Mockito.verify(safeUserStorageWrapper, Mockito.times(1)).set(Mockito.eq("SPLITIO.telemetry.init::SDK 4.2.x/testMachine/22.215135.1"), Mockito.anyObject());
+    }
+
+    @Test
+    public void testTestSynchronizeUniqueKeys() throws NoSuchFieldException, IllegalAccessException {
+        SafeUserStorageWrapper safeUserStorageWrapper = Mockito.mock(SafeUserStorageWrapper.class);
+        TelemetryConsumerSubmitter telemetrySynchronizer = new TelemetryConsumerSubmitter(Mockito.mock(CustomStorageWrapper.class), new SDKMetadata("SDK 4.2.x", "22.215135.1", "testMachine"));
+        Field telemetryConsumerSubmitterHolder = TelemetryConsumerSubmitter.class.getDeclaredField("_safeUserStorageWrapper");
+        telemetryConsumerSubmitterHolder.setAccessible(true);
+        telemetryConsumerSubmitterHolder.set(telemetrySynchronizer, safeUserStorageWrapper);
+
+        List<String> keys = new ArrayList<>();
+        keys.add("key-1");
+        keys.add("key-2");
+        List<UniqueKeys.UniqueKey> uniqueKeys = new ArrayList<>();
+        uniqueKeys.add(new UniqueKeys.UniqueKey("feature-1", keys));
+        UniqueKeys uniqueKeysToSend = new UniqueKeys(uniqueKeys);
+
+        telemetrySynchronizer.synchronizeUniqueKeys(uniqueKeysToSend);
+        Gson gson = new GsonBuilder().create();
+        List<String> uniqueKeysJson = new ArrayList<>(Arrays.asList(gson.toJson(uniqueKeysToSend)));
```

Comment:
nit: Creating the `Gson` instance may be avoided by using the Json string directly.
```suggestion
        List<String> uniqueKeysJson = new ArrayList<>(Collections.singletonList("{\"keys\":[{\"f\":\"feature-1\",\"ks\":[\"key-1\",\"key-2\"]}]}"));
```

Reply:
I updated the test case.